### DISCUSSION TITLE
Sistema de drag and drop avançado

### DISCUSSION THREAD

**Discussion by pedroestevaodev** [GrapesJS/grapesjs#3784] [Ideas]
Gostaria de saber se existe ou se alguém está desenvolvendo um plugin que permita movimentar os blocos livremente pelo uso de coordenadas X e Y, que elimine a necessidade de ter que ficar definindo margens e orientações.

**Comment by artf** [comment]
English, please

**Comment by pedroestevaodev** [comment]
I would like to know if there is or if someone is developing a plugin that allows you to move blocks freely by using X and Y coordinates, which eliminates the need to keep defining margins and orientations.

**Comment by Ju99ernaut** [comment]
There's already a built-in solution https://github.com/artf/grapesjs/issues/1936
